easybuild/easyconfigs/b/bx-python/bx-python-0.8.2-foss-2018b-Python-3.6.6.eb

```diff
@@ -0,0 +1,43 @@
+easyblock = 'PythonBundle'
+
+name = 'bx-python'
+version = '0.8.2'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://github.com/bxlab/%(name)s'
+description = """The bx-python project is a Python library and associated set of scripts to allow for rapid
+ implementation of genome scale analyses."""
+
+toolchain = {'name': 'foss', 'version': '2018b'}
+
+dependencies = [
+    ('Python', '3.6.6'),
+    ('LZO', '2.10'),
+]
+
+prebuildopts = "export CPATH=$EBROOTLZO/include/lzo:$CPATH && "
+
+use_pip = True
+
+exts_download_dep_fail = True
+
+exts_list = [
+    ('python-lzo', '1.12', {
+        'modulename': 'lzo',
+        'source_urls': ['https://pypi.python.org/packages/source/p/python-lzo/'],
+        'checksums': ['97a8e46825e8f1abd84c2a3372bc09adae9745a5be5d3af2692cd850dac35345'],
+    }),
+    (name, version, {
+        'modulename': 'bx',
+        'source_urls': ['https://pypi.python.org/packages/source/b/%(name)s/'],
+        'checksums': ['faeb0c7c9fcb2f95c4fc1995af4f45287641deee43a01659bd30fe95c5d37386'],
+    }),
+]
+
+sanity_check_paths = {
+    'files': [],
+    'dirs': ['bin', 'lib/python%(pyshortver)s/site-packages/bx_python-%(version)s-py%(pyshortver)s.egg-info',
+             'lib/python%(pyshortver)s/site-packages/python_lzo-1.12-py%(pyshortver)s.egg-info'],
```

Comment:
@SethosII We usually only check for `'lib/python%(pyshortver)s/site-packages'` here, more specific checks are prone to get broken (in particular when versions of required dependencies are hardcoded like you did here for `python-lzo`)

Reply:
@boegel The intention was to check that both packages of the bundle are there. Is there a way to use the `exts_list` variable for that? I removed it for now.

Reply:
EasyBuild already does an `import` check for each extension. There have been cases where Python packages get installed slightly differently under some circumstances (e.g. as a zipped `.egg` *file* vs an (unzipped) `.egg` directory), which is why we've opted for only checking the parent directory.

In theory the `import` could still work if an extension accidentally was installed somewhere else instead, but we can look into enhancing the `import` check to catch those too (by checking where the module was imported from).